client/actions/getFoodDetailsAction.test.js

```diff
@@ -0,0 +1,55 @@
+import { getFoodDetailsRequest, getFoodDetailsReceived, getFoodDetails } from './getFoodDetailsActions'
+
+const mockFoods = [
```

Comment:
Would make more sense if this was one single food, to better mimic the actual use of the action.